docs/why.qmd

```diff
@@ -2,24 +2,224 @@
 title: "Why Ibis?"
 ---
 
+Ibis defines a Python dataframe API that supports any query engine -- the
+frontend for any backend data platform. This includes 20+ backends today, most
+SQL-based.  This allows Ibis to have excellent performance -- as good as the
+backend it is connected to -- with a consistent user experience.
+
```

Comment:
Would this page be a good place for the harder Ibis sell? Something like this:

What if you didn't have to learn yet another query language to try out a new database? What if moving from one analytical system to another didn't require you to rewrite all of your existing pipelines? What if you could just use the familiar, pandas-like DataFrame API you already use every day to do all of this and more?

Ibis solves these problem today for more than 20 popular database backends and the list will continue to grow. On the fly, Ibis translates your code to the query language of the backend you choose which means you get exactly the performance your database advertises without sacrificing the ability make another choice of backend in the future.

Reply:
yes! let's do this in a follow up and combine w/ Z's comment below -- I think it should be a collapsed callout block where we go into a more detailed "sales pitch" on Ibis

Reply:
https://github.com/ibis-project/ibis/issues/8514

Reply:
Sounds great, thanks